src/tmux.ts

```diff
@@ -52,12 +53,22 @@ export function setShellConfig(config: { type: string }): void {
   }
 }
 
+export function setSSHPrefix(conn_str: string): void {
```

Comment:
Maybe `connectionString` is more idiomatic 🤓 

```suggestion
export function setSSHPrefix(connectionString: string): void {
```